tamr_unify_client/models/binning_model.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+
+class BinningModelObject:
+    """ A binning model object."""
+
+    def __init__(self, client, api_path="projects/1/binningModel"):
```

Comment:
Remove default value for `api_path`

Reply:
Wouldn't that default value be necessary? It doesn't seem to run otherwise.